Makefile

```diff
@@ -13,24 +13,51 @@ no_test: programs
 
 programs: lib
 	$(MAKE) -C programs
+ifdef ENABLE_PSA
+	$(MAKE) -C crypto/programs
```

Comment:
As suggested above, I think we can remove this.

Reply:
Removed